app/assets/stylesheets/limber/screen.scss

```diff
@@ -182,6 +182,9 @@ dl.metadata, dl#samples-information {
   input[type=checkbox]:checked + label .aliquot {
     background-color: red;
   }
+  input[type=checkbox]:disabled + label .aliquot {
+    opacity: 0.5;
+  }
 
```

Comment:
In order to provide a bit of visual feedback that some wells aren't failable, we fade them out slightly.